precisely/comparison_matchers.py

```diff
@@ -45,3 +46,22 @@ def match(self, actual):
 
     def describe(self):
         return "{0} {1!r}".format(self._operator_description, self._value)
+
+
+def close_to(value, delta):
+    return IsCloseToMatcher(value, delta)
+
+
+class IsCloseToMatcher(object):
+    def __init__(self, value, delta):
+        self._value = value
+        self._delta = delta
+
+    def match(self, actual):
+        if fabs(actual - self._value) <= self._delta:
```

Comment:
Is there a reason to use `fabs` rather than `abs`?

Come to think of it, avoiding `abs` might be useful to avoid unnecessarily not supporting other data types. For instance, you could imagine wanting to do a similar operation using datetimes.

Perhaps the check could be `value - delta <= actual <= value + delta`?

Reply:
Hamcrest use `fabs` - I assume so that the matcher does *only* support numeric data types (it's in `hamcrest.library.number`). I agree it's better to remove that limitation.

Reply:
Further - I'll move the tests out of `numeric_comparison_tests.py` (into `comparison_tests.py` ?) and add a `datetime` test.